### PR TITLE
Follow Log Example.

### PR DESCRIPTION
**Pull request by hardillb**
As part of mitigation for #689 here is an example of how to use the Log class to follow logs.

### CODE REVIEW

**Comment by hardillb** [comment]
CLA signed

File: examples/follow-logs.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+kc.loadFromDefault()
+
+const log = new k8s.Log(kc)
+
+const logStream = new stream.PassThrough();
+
+logStream.on('data', (chunk) => {
+	// use write rather than console.log to prevent double line feed
+	process.stdout.write(chunk);
+})
```
Comment (brendandburns) [inline examples/follow-logs.js:14]: semi-colon here.

File: examples/follow-logs.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+log.log('default', 'pod1', 'container1', logStream, (err) => {console.log(err)}, {follow: true, tailLines: 50, pretty: false, timestamps: false})
+.then(req => {
+	// disconnects after 5 seconds
+	setTimeout(function(){
+		req.abort()
+	}, 5000)
+})
```
Comment (brendandburns) [inline examples/follow-logs.js:22]: semi-colon here.

File: examples/follow-logs.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+logStream.on('data', (chunk) => {
+	// use write rather than console.log to prevent double line feed
+	process.stdout.write(chunk);
+})
+
+log.log('default', 'pod1', 'container1', logStream, (err) => {console.log(err)}, {follow: true, tailLines: 50, pretty: false, timestamps: false})
+.then(req => {
+	// disconnects after 5 seconds
+	setTimeout(function(){
+		req.abort()
```
Comment (brendandburns) [inline examples/follow-logs.js:20]: semi-colon here.

File: examples/follow-logs.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	}, 5000)
```
Comment (brendandburns) [inline examples/follow-logs.js:21]: here too.

**Comment by brendandburns** [comment]
Thanks for the PR! Please add semi-colons to terminate all lines and then I think this is ok to merge.

**Comment by brendandburns** [comment]
Thanks for the quick turn-around.

(note to self, we should validate that our style/lint checks run on examples also)

/lgtm
/approve

**Pull request merged**